environmentmodel/environment-model_shacl.ttl

```diff
@@ -0,0 +1,288 @@
+@prefix environmentmodel: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/environmentmodel/> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+environmentmodel:environmentModelShape a sh:NodeShape ;
```

Comment:
Shape in PascalCase ->EnvironmentModelShape

Reply:
I adapted that.